libraries/standard/mqtt/src/mqtt_state.c

```diff
@@ -964,7 +959,8 @@ MQTTStatus_t MQTT_UpdateStatePublish( MQTTContext_t * pMqttContext,
  * @param[in] opType Send or Receive.
  * @param[out] pNewState Updated state of the publish.
  *
- * @return #MQTTBadParameter, #MQTTIllegalState, or #MQTTSuccess.
+ * @return #MQTTBadParameter, #MQTTBadResponse, #MQTTIllegalState, or
+ * #MQTTSuccess.
  */
```

Comment:
Why is there doxygen documentation for these public facing functions inside the source file?

Reply:
It was added in #1120: https://github.com/aws/aws-iot-device-sdk-embedded-C/pull/1122#discussion_r470146719